### ISSUE TITLE
Fix rubocop warning of wrong namespace

### ISSUE THREAD

**Issue by KrauseFx** [fastlane/ci#509]
```
.rubocop.yml: Style/MultilineMethodCallIndentation has the wrong namespace - should be Layout
```
Happens when running `be rubocop -a`, maybe it's related to my Ruby version?

**Comment by adellibovi** [comment]
Not seeing this using 2.5.1

**Comment by AndrewMcBurney** [comment]
I think this was fixed by #511. More specifically [this line](https://github.com/fastlane/ci/pull/511/files#diff-11a0d7906801d9dea0eccb85667ad811R58).

Is it cool to close this issue?

**Comment by KrauseFx** [comment]
Yeah, let's close it, it's already been 9 days, which is 3 years in non fastlane.ci time 👍
